### PR TITLE
Add doc for importing Ansible roles

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by tstrachota** [comment]
FYI I added PR https://github.com/xprazak2/theforeman.org/pull/1 it would be good to merge them together.

**Comment by dLobatog** [comment]
I'll wait until we make a new release to merge this, it doesn't make sense for the current published release. Thanks @xprazak2 !

**Comment by xprazak2** [comment]
I merged in @tstrachota's PR describing the assignment of roles through hostgroups

File: plugins/foreman_ansible/1.x/index.md
Comment (dLobatog) [inline plugins/foreman_ansible/1.x/index.md]: `/etc/ansible/roles` not `hosts`. Instead of from your 'local' , I'd specify from your 'Foreman host', it's less ambiguous.

`You can import hosts from the dropdown` - and add a screenshot  

typos
- 'List of all roles`->`A list of all roles`

File: plugins/foreman_ansible/1.x/index.md
Comment (dLobatog) [inline plugins/foreman_ansible/1.x/index.md]: Capitalize `Ansible`

File: plugins/foreman_ansible/1.x/index.md
Comment (dLobatog) [inline plugins/foreman_ansible/1.x/index.md]: `The host group edit form` , also capitalize `Ansible`

File: plugins/foreman_ansible/1.x/index.md
Comment (dLobatog) [inline plugins/foreman_ansible/1.x/index.md]: `host group`

**Review by dLobatog — CHANGES_REQUESTED** [review]
Please take a look, thanks @xprazak2 !

**Comment by xprazak2** [comment]
I fixed the capitalization, typos and added a dropdown screenshot.

**Review by dLobatog — APPROVED** [review]

File: plugins/foreman_ansible/1.x/index.md
Comment (dLobatog) [inline plugins/foreman_ansible/1.x/index.md:100]: hostgroup -> host group. but I'll change that on merge

**Comment by dLobatog** [comment]
Merged as 29b87f86404144ca5bc7acca1626dfb9c6a9a19b , thanks @xprazak2  and @tstrachota !

**Pull request closed without merging**
